src/message/index.js

```diff
@@ -97,25 +97,18 @@ Message.TYPES = MESSAGE_TYPE
 Message.CONNECTION_TYPES = CONNECTION_TYPE
 
 function toPbPeer (peer) {
-  const res = {
+  return {
     id: peer.id.id,
-    addrs: peer.multiaddrs.toArray().map((m) => m.buffer)
+    addrs: (peer.multiaddrs || []).map((m) => m && m.buffer),
```

Comment:
What's the reasoning for the check on m (`m && m.buffer`)? If it's possible for `m` to be null or undefined we should do a reduce, otherwise we'll end up with undefined in the `addrs` array for that value.

Reply:
It was me when trying to debug a bug to short circuit it. But it is not needed anymore 😆